src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheckTest.java

```diff
@@ -340,13 +342,26 @@ public void testExcludeScope()
             "315: " + getCheckMessage(MSG_NO_PERIOD),
             "321:11: " + getCheckMessage(MSG_UNCLOSED_HTML,
                 "<b>Note:<b> it's unterminated tag.</p> // violation"),
-            "336: " + getCheckMessage(MSG_NO_PERIOD),
```

Comment:
Question:
So this class is truely package private since it is inside a class?
Why did your changes impact this?

How come your changes do not need to account for class/interface? Taking the code from the issue, changing interface to class changes the implied modifier for the nested class.

Reply:
The earlier logic of mine was wrong, so there were a lot of wrong violations. Fixed it now.